doc/source/whatsnew/v0.25.0.rst

```diff
@@ -285,7 +285,7 @@ Timedelta
 
 - Bug in :func:`TimedeltaIndex.intersection` where for non-monotonic indices in some cases an empty ``Index`` was returned when in fact an intersection existed (:issue:`25913`)
 - Bug with comparisons between :class:`Timedelta` and ``NaT`` raising ``TypeError`` (:issue:`26039`)
--
+- Bug in adding of business hour to the timestamp with an ending time landing in the next business time period (:pull:`26381`)
```

Comment:
``Bug when adding or subtracting a :class:`BusinessHour` to a :class:`Timestamp` with the resulting time landing in the next or prior day respectively (:issue:`26381`)``